stable/insights-agent/templates/right-sizer/controller-deployment.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.rightsizer.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "insights-agent.fullname" . }}-right-sizer-controller
+  annotations:
+  labels:
+    component: right-sizer
+    app: insights-agent
+spec:
+  selector:
+    matchLabels:
+      component: right-sizer
+      app: insights-agent
+  template:
+    metadata:
+      labels:
+        component: right-sizer
+        app: insights-agent
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}-right-sizer-controller
+          ports:
+          - containerPort: 8080
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
```

Comment:
mmh do we have `Values.securityContext defined any where? and i would assume if they were to be defined should be in under `Values.rightsizer.securityContext`

Reply:
Thanks - moved in the deployment template values specification, and defined in values also.